keyboards/romac/keymaps/default/keymap.c

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2018 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "romac.h"
+
+#define _BASE 0
+#define _FN1 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[_BASE] = LAYOUT(
```

Comment:
Your layout macro here (LAYOUT) doesn't match what you defined in the `.h` file (KEYMAP). These two need to match. I would recommend changing KEYMAP to LAYOUT.

Reply:
Did I resolve this?  I committed changes but this review still stands.  Thank you for dealing with all my ignorance

Reply:
You didn't actually need to change anything here, because the issue was in another file.

I'll take another pass through the code and look for any outstanding issues.